src/omv/imlib/imlib.h

```diff
@@ -385,7 +417,10 @@ bool image_get_mask_pixel(image_t *ptr, int x, int y);
     (_image->bpp == IMAGE_BPP_BINARY) || \
     (_image->bpp == IMAGE_BPP_GRAYSCALE) || \
     (_image->bpp == IMAGE_BPP_RGB565) || \
-    (_image->bpp == IMAGE_BPP_BAYER); \
+    (_image->bpp == IMAGE_BPP_BAYER_BGGR) || \
+    (_image->bpp == IMAGE_BPP_BAYER_GBRG) || \
+    (_image->bpp == IMAGE_BPP_BAYER_GRBG) || \
+    (_image->bpp == IMAGE_BPP_BAYER_RGGB); \
 })
 
 #define IMAGE_IS_MUTABLE_BAYER_JPEG(image) \
```

Comment:
@kwagyeman What's this macro for ? it's basically always true.

Reply:
Not for jpeg. But, it depends on where it's used.

Reply:
https://github.com/openmv/openmv/blob/master/src/omv/imlib/imlib.h#L391

It literally tests all the formats.